pkg/s3/s3_service.go

```diff
@@ -86,10 +86,11 @@ func (ps3 *S3Client) ListBuckets(ctx context.Context) (*BucketLists, error) {
 	return &res, nil
 }
 
-func (ps3 *S3Client) ListFiles(ctx context.Context, bucket *string) (*FileLists, error) {
+func (ps3 *S3Client) ListFiles(ctx context.Context, bucket, prefix *string) (*FileLists, error) {
```

Comment:
I don't think this is used anywhere at this point.
https://github.com/search?q=org%3Aphil-inc+ListFiles&type=code

Reply:
@grace-punzalan  Maybe create a new function instead as prefix might not be required in some cases, or maybe check if nil prefix is supported?

Reply:
I tested that an empty string prefix returns all the files in the bucket. Is that enough? If not I can create a new function too.

Reply:
@grace-punzalan Then I think we can modify this function itself as it supports both purposes and isn't used anywhere at this point.